packages/edit-post/src/components/options-modal/test/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { OptionsModal } from '../';
+
+describe( 'OptionsModal', () => {
+	it( 'should match snapshot when the modal is active', () => {
+		const wrapper = shallow( <OptionsModal isModalActive={ true } /> );
+		expect( wrapper ).toMatchSnapshot();
+	} );
+
+	it( 'should match snapshot when the modal is not active', () => {
+		const wrapper = shallow( <OptionsModal isModalActive={ false } /> );
+		expect( wrapper ).toMatchSnapshot();
```

Comment:
Minor: It might be easier to read when you check in the test if it renders nothing instead. Snapshots are a bit indirect as you need to navigate to another file.

Reply:
👌 11f7c2d55